depends/patches/qt/macos_skip_version_checks.patch

```diff
@@ -0,0 +1,69 @@
+The @available compiler directive exports the __isPlatformVersionAtLeast
+symbol, which the linker cannot resolve during cross-compilation.
+
+This patch assumes that the minimum supported macOS version is 13.0.
```

Comment:
In In 9d3be36f758095167ddd762400fda1a3ea76e8f9:

> symbol, which the linker cannot resolve during cross-compilation.

Which linker doesn't this work with? I dropped this patch, and cross-compiled for macOS on Ubuntu, with lld `18.1.3`, and it seemed to work fine? It'd be good if the patch could include the details.

Reply:
Thanks! It works for me too, so I've dropped the patch. It was possibly a side effect of https://github.com/bitcoin/bitcoin/pull/31048.